### PR TITLE
Improve quota messages for clarity and conciseness

### PR DESCRIPTION
**Pull request by bpasero**

### CODE REVIEW

File: src/vs/workbench/contrib/chat/browser/chatQuotasService.ts [hunk 1]
```diff
@@ -264,10 +264,10 @@ export class ChatQuotasStatusBarEntry extends Disposable implements IWorkbenchCo
 
 export function quotaToButtonMessage({ chatQuotaExceeded, completionsQuotaExceeded }: { chatQuotaExceeded: boolean; completionsQuotaExceeded: boolean }): string {
 	if (chatQuotaExceeded && !completionsQuotaExceeded) {
-		return localize('chatQuotaExceededButton', "You've reached your monthly chat messages limit, click for details");
+		return localize('chatQuotaExceededButton', "Monthly chat messages limit reached, click for details");
```
Comment (youneshenniwrites) [inline src/vs/workbench/contrib/chat/browser/chatQuotasService.ts:267]: I think the grammatically correct version should be:

`Monthly chat message limit reached. Click for details.`

**Review by justschen — APPROVED** [review]

**Comment by justschen** [comment]
oh woops, I saw the comment by @youneshenniwrites but hit approve on accident 💀 sorry about that @bpasero

File: src/vs/workbench/contrib/chat/browser/chatQuotasService.ts (hunk 1 above)
Comment (bpasero) [reply src/vs/workbench/contrib/chat/browser/chatQuotasService.ts:267]: Yes sir

Comment (bpasero) [reply src/vs/workbench/contrib/chat/browser/chatQuotasService.ts:267]: Sounds good, thanks!

**Pull request merged**